testsuite/MDAnalysisTests/lib/test_cutil.py

```diff
@@ -37,6 +40,10 @@
     [1, 2, 2, 4, 4, 6, ],  # duplicates, monotonic
     [1, 2, 2, 6, 4, 4, ],  # duplicates, non-monotonic
 ))
+
+def u():
+    return mda.Universe(PSF, DCD)
+
```

Comment:
remove, when testing `lib` functions just use normal arrays because all these functions work without universes and AtomGroups